new_project/examples/separate_apps_testnet/README.md

```diff
@@ -0,0 +1,219 @@
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api.socket]
+address = "0.0.0.0"
+port = 18000
+
+[http_api.cors]
+allowed_origins = "all"
+
+[Data]
+dir = "~/comit/maker"
+```
+
+Create a seed file for the taker called `maker_seed.pem`:
+```
+-----BEGIN SEED-----
+AaBZdOwqLWHTLe9HXEc//wVGdt4VzVkDB/jB7tzkf/c=
+-----END SEED-----
+```
+
+Startup command:
+```
+cnd --config maker_config.toml --seed-file maker_seed.pem
+```
+
+### Key/account generation and funding for both wallets
+
+Keep in mind that when running both maker and taker on the same machine you have to create a Bitcoin and Ethereum account for both maker and taker, thus you have to create and fund **two Bitcoin** and **two Ethereum** accounts.
+
+Configuration in `env` file (assuming that you run on both maker and taker from the same machine):
+
+```
+BITCOIN_HD_KEY_TAKER=EXTENDED_PRIVATE_KEY_GENERATED_BY_WAGYU
+ETHEREUM_KEY_TAKER=ETHEREUM_PRIVATE_KEY_GENERATED_BY_WAGYU
+
+BITCOIN_HD_KEY_MAKER=EXTENDED_PRIVATE_KEY_GENERATED_BY_WAGYU
+ETHEREUM_KEY_MAKER=ETHEREUM_PRIVATE_KEY_GENERATED_BY_WAGYU
+
+HTTP_URL_CND_TAKER=http://localhost:8000
+HTTP_URL_CND_MAKER=http://localhost:18000
+
+BITCOIN_P2P_URI=bobtimus.coblox.tech:18332
+ETHEREUM_NODE_HTTP_URL=http://bobtimus.coblox.tech:8545
+```
+
+#### Bitcoin
+
+1. Generate HD key (master-key) with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu bitcoin hd --network testnet --derivation "m"`
+2. Copy the `Extended Private Key` generated by Wagyu into the `env` file.
+3. Generate an address to fund with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu bitcoin import-hd --derivation "m/44'/1'/0'/0/0" --format bech32 --extended-private GENERATED-EXTENDED-PRIVATE-KEY`
+    2. Fund the `Address` returned
+4. Fund the address through a [Bitcoin testnet coin faucet](https://coinfaucet.eu/en/btc-testnet/)
+
+#### Ethereum
+
+1. Generate an Ethereum account with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu ethereum hd`
+2. Copy the `Private Key` (NOT the extended private key) to the env file.
+3. Fund the account through an [Ethereum ropsten coin faucet](https://faucet.metamask.io/)
+
+
+## Run the example
+
+To ensure that the swap works (assuming that you run maker and taker on one machine):
+1. Both maker and taker cnd are running,
+2. `env` file was properly initialized with the bitcoin HD keys and Ethereum private keys
+3. Bitcoin addresses and Ethereum accounts were properly funded.
+
+
+TO BE REMOVED:
+* Start example, print balances of addresses, addresses are printed "Please fund: ..."
+* If you don't have sufficient funds you may have to refund through a coin faucet.
+
+
+## Appendix
+
+### CoBloX Testnet Node Status
+
+#### Ethereum
+```
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' -H "Content-type:application/json" bobtimus.coblox.tech:8545 | jq .
```

Comment:
`bobtimus.coblox.tech` is pretty outdated, isn't it? If we want to provide these machines for real, I would suggest to use more descriptive hostnames (`bitcoind-testnet.coblox.tech`, `parity-testnet.coblox.tech`?)

Reply:
I would rather not provide it to the public and rather just use them when we need them, e.g. for workshops/presentations :)

Reply:
> I would rather not provide it to the public and rather just use them when we need them, e.g. for workshops/presentations :)

I totally agree with that statement. My point still stands: the hostname is outdated we should be changed accordingly.